src/services/email/email.definition.ts

```diff
@@ -0,0 +1,9 @@
+export interface EmailInfo {
+  to: Array<string>;
+  from: string;
```

Comment:
emailService should fill this in with whatever email we're sending from

Reply:
aka as a developer it would be nice not to have the hkn email in multiple places around the codebase

Reply:
Well actually, 'from' here only defines what you need to pass into the emailInfo object. As in the object needs to have 'from' (SendGrid requires 'from' as well, so we'll need to have it). What we can do is just to have an env var or something for the hkn email, then pass it into 'from' when we call the public email functions in the future.

Reply:
in general it's good practice to load env vars only once - should load it in email.service and hide it from the user

Reply:
Yeah that's fine as well. I'll do that. Though we will have to keep 'from' property's type definition.

Reply:
why?

Reply:
Because SendGrid's send() function requires it. Our custom interface has to have it or else it will give an error. I have tried taking it out already, believe me. And if I do take 'from' out, and try to insert 'from' into the EmailInfo objects, then it will also give an error, so there is no way around it.

Reply:
ah gotcha - not sure why reinserting from doesn't work but I'll take your word on that

Reply:
I can demo tomorrow if you would like. Anything else you want me to fix? I think this should be ready for merging now.

Reply:
Also, I think reinserting 'from' doesn't work because once I have declared the specific custom EmailInfo type for the params, then it uses that to match types when u pass the EmailInfo object as a param to SendGrid's send(). Meaning that even if you reinsert 'from', its type still doesn't have a 'from' property so it gives an error.